tests/test_norm.py

```diff
@@ -0,0 +1,159 @@
+import torch
+import torch_ttnn
+import ttnn
+import unittest
+
+
+class TestModules(unittest.TestCase):
```

Comment:
@yocosky , this is not following the established pattern

See example
https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/tests/lowering/tensor_manipulation/test_permute.py

We are using pytest
* No need to duplicate setup/teardown everywhere
* easy way to parametrize tests
* clean way to define whats tested
* clean way to check behavior